staging/src/k8s.io/legacy-cloud-providers/azure/azure_routes.go

```diff
@@ -487,6 +489,28 @@ func (az *Cloud) DeleteRoute(ctx context.Context, clusterName string, kubeRoute
 		return err
 	}
 
+	// Remove outdated ipv4 routes as well
+	if az.ipv6DualStackEnabled {
+		routeNameWithoutIPV6Suffix := strings.Split(routeName, routeNameSeparator)[0]
```

Comment:
What makes this "WithoutIPV6Suffix"? It's using the same routeName variable as above isn't it?

Reply:
Ah, is it cause even on dual-stack the azure provider only supports IPv4 routes, so it's just a matter of parsing a different naming format when dual-stack is enabled??

Reply:
I want to make sure the outdated ipv4 routes are deleted if enabled dualstack.

Reply:
I guess my question earlier was asking: how do you know this route is the IPv4 without checking the CIDR? Is that done somewhere else?

Reply:
The naming conventions are different for ipv4 and ipv6 routes.